gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/CreateOrUpdateRecordSets.java

```diff
@@ -23,6 +23,7 @@
 package com.google.gcloud.examples.dns.snippets;
 
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
I believe the `ChangeRequest` import is no longer needed. Might be wrong though.